airbyte-integrations/bases/base-normalization/normalization/transform_catalog/transform.py

```diff
@@ -169,7 +169,7 @@ def json_extract_base_property(path: List[str], json_col: str, name: str, defini
         return None
     elif is_string(definition["type"]):
         return "cast({} as {}) as {}".format(
-            jinja_call(f"json_extract('{json_col}', {current})"),
+            jinja_call(f"json_extract_scalar('{json_col}', {current})"),
```

Comment:
FYI @ChristopheDuong - I think this is the correct method to call but please correct me otherwise

Reply:
It was actually using `json_extract_scalar` for strings before and had to change it to at some point `json_extract` otherwise i saw NULL values in some cases even though i was expecting some values...

Is there a specific reason you need to change it back?

Reply:
BigQuery was double-quoting strings without this change. E.g: "2020-10-20" was written as "\"2020-10-20\"" in the destination